zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1423,21 +1423,99 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /variables:
+    post:
+      tags:
+        - Variable
+      operationId: createVariable
+      summary: Create variable
+      description: |
+        Create cluster level variables
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateVariableRequest"
+      responses:
+        "201":
+          description: >
+            The variable creation status.
```

🔧 We know the result, let's describe it like this.

```suggestion
            The variable was created successfully.
```